Homework1/Models/Person.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+
+namespace Homework1.Models
+{
+    public enum Gender
+    {
+        Male,
+        Female
+    }
+
+    public class Person
+    {
+        #region Public Properties
```

Comment:
Don't use regions- it's an anti-pattern. It actually makes code harder to read.

Reply:
those come from the codemaid extension... haven't set it up correctly